### PR TITLE
Updates to gpu tech note

### PR DESCRIPTION
**Pull request by stonea**
- Use here.gpus[] instead of here.getChild().
- Add section on multilocale support

### CODE REVIEW

File: doc/rst/technotes/gpu.rst [hunk 1]
```diff
@@ -63,7 +58,8 @@ there.  This may also be set using the ``CHPL_CUDA_ARCH`` environment variable.
 If you would like to view debugging information you can pass ``--verbose`` to
 your generated executable. This output will show the invocation of CUDA kernel
 calls along with various other interactions with the GPU such as memory
-operations.
+operations.  You may also use the GPUDiagnostics module to gather similar
```
Comment (e-kayrakli) [inline doc/rst/technotes/gpu.rst:61]: Can you add a link to the module documentation? Grepping `:ref:` in the technotes can lead to the right incantation for it.

File: doc/rst/technotes/gpu.rst [hunk 2]
```diff
@@ -99,3 +95,27 @@ kernels for the ``forall`` loops in the function.
...
 
 
+Multi-Locale Support
+--------------------
+
+As of Chapel 1.27.0 the GPU locale model may be used alongside communication
+layers (values of ``CHPL_COMM``) other than ``none``. This enables programs to
+use of GPUs across nodes.
+
+In this mode, normal inter-node PGAS style access is supported outside of loops
```
Comment (e-kayrakli) [inline doc/rst/technotes/gpu.rst:105]: Use "remote access" instead of "inter-node PGAS style" access?

**Review by e-kayrakli — APPROVED** [review]

File: doc/rst/technotes/gpu.rst (hunk 1 above)
Comment (stonea) [reply doc/rst/technotes/gpu.rst:61]: Updated; this is how you do it BTW: ``:mod:`GPUDiagnostics` ``

File: doc/rst/technotes/gpu.rst (hunk 2 above)
Comment (stonea) [reply doc/rst/technotes/gpu.rst:105]: Sure, updated

**Pull request merged**